client/__tests__/reducers/cascade.test.ts

```diff
@@ -1,3 +1,6 @@
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment -- okay to disable for tests
+// @ts-nocheck -- Typing getting in way of unit test
```

Comment:
The test here tests the cascading reducer correctly, but doesn't quite conform to the typing. Not sure if its worth fixing the typing/test to align at this time.